pkg/leakybucket/bucket.go

```diff
@@ -256,7 +259,10 @@ func LeakRoutine(leaky *Leaky) {
 			if leaky.timedOverflow {
 				BucketsOverflow.With(prometheus.Labels{"name": leaky.Name}).Inc()
 
-				alert = NewAlert(leaky, ofw)
+				alert, err := NewAlert(leaky, ofw)
```

Comment:
there's a scope issue here
alert is not the same as the alert out of the if check.

Reply:
@registergoofy can you be more explicit ?